### ISSUE TITLE
Changer le label de hasName

### ISSUE THREAD

**Issue by mrivoal** [LaDHUL/Theatre-de-societe#41]

**Comment by mrivoal** [comment]
```
###  http://www.knora.org/ontology/0103/theatre-societe#hasName
theatre-societe:hasName rdf:type owl:ObjectProperty ;
         rdfs:subPropertyOf knora-base:hasValue ;
         knora-base:objectClassConstraint knora-base:TextValue ;
         rdfs:label "Prénom"@fr ,
					"Given Name"@en ;
		 rdfs:comment "Prénom"@fr ,
					"The person's given name"@en .
```

**Comment by mrivoal** [comment]
Closed via 282f586cd6619f03ae25d764967a7f7ef95b1359
